tasks/17_truncate-string/truncate-string.js

```diff
@@ -0,0 +1,8 @@
+export function truncateString(str, maxLength) {
+  if (str.length > maxLength && maxLength > 3) {
+    return str.slice(0, maxLength - 3) + '...';
+  } else if (maxLength < 3) {
+    return str.slice(0, maxLength) + '...';
+  }
+  return str;
```

Comment:
If `'some string'` and `3` were passed to your function, can you guess the outcome. Luckily or unfortunately there is no test to cover this case, but still this is something that is described in the readme of the task.

Also, I would open this if statement with `if (maxLength < =3) {`

Reply:
Again you are absolutely right, I fixed the code and refactored it based on your suggestion